### PR TITLE
Update the warning message after running variant analysis

### PR DESCRIPTION
**Pull request by aeisenberg**
Adds more information about onboarding new repos.

### CODE REVIEW

**Review by shati-patel — APPROVED** [review]
🚢

File: extensions/ql-vscode/src/remote-queries/run-remote-query.ts [hunk 1]
```diff
@@ -353,6 +353,7 @@ export function parseResponse(owner: string, repo: string, response: QueriesResp
     }
     if (response.errors.repositories_without_database?.length) {
       logMessage += `\n\nRepositories without databases:\n${response.errors.repositories_without_database.join(', ')}`;
+      logMessage += '\nThese repositories have been added to the database storage service and we will attempt to create a database for them next time the store is updated.';
```
Comment (charisk) [inline extensions/ql-vscode/src/remote-queries/run-remote-query.ts:356]: Should we be using `os.EOL` instead of `\n`?

Comment (aeisenberg) [reply extensions/ql-vscode/src/remote-queries/run-remote-query.ts:356]: Sure. I'll make that change.

File: extensions/ql-vscode/src/remote-queries/run-remote-query.ts [hunk 2]
```diff
@@ -338,22 +339,25 @@ async function runRemoteQueriesApiRequest(
 }
 
 // exported for testng only
```
Comment (charisk) [inline extensions/ql-vscode/src/remote-queries/run-remote-query.ts:341]: Might want to move the comment to be on top of the function.

**Review by charisk — APPROVED** [review]

File: extensions/ql-vscode/src/remote-queries/run-remote-query.ts [hunk 3]
```diff
@@ -338,22 +339,25 @@ async function runRemoteQueriesApiRequest(
...
+const eol = os.EOL;
+const eol2 = os.EOL + os.EOL;
+
 export function parseResponse(owner: string, repo: string, response: QueriesResponse) {
```
Comment (aeisenberg) [inline extensions/ql-vscode/src/remote-queries/run-remote-query.ts:345]:
```suggestion
const eol = os.EOL;
const eol2 = os.EOL + os.EOL;

// exported for testng only
export function parseResponse(owner: string, repo: string, response: QueriesResponse) {
```

**Pull request merged**